README.md

```diff
@@ -28,7 +28,8 @@ UPSTREAM_CHE_PATH=/path/to/upstream/che
 mvn -DwithoutKeycloak                          `# disable Keycloak support` \
     -Pfast                                     `# skip tests and other verifications` \
     -PmultiThread                              `# enable maven multi threading` \
-    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che` \
+    -DlocalCheRepository=${UPSTREAM_CHE_PATH}  `# get already built upstream Che local folder` \ 
```

Comment:
explained in comment https://github.com/redhat-developer/rh-che/pull/77#discussion_r118252410

> when you build with `localCheRepository`, it only builds the RH distribution but, when it needs to __*access the upstream Che sources (e.g. GWT application or Dashboard for example)*__, or the upstream Che generated artifacts (e.g. __*assembly-main*__), it will get them from the local upstream Che repository defined by the `localCheRepository` setting.

The RH build needs need both generated artifacts __*and*__ the upstream __*sources*__ according to the modules.